cms/envs/acceptance.py

```diff
@@ -127,3 +127,7 @@ def seed():
 # Generate a random UUID so that different runs of acceptance tests don't break each other
 import uuid
 SECRET_KEY = uuid.uuid4().hex
+
+############################### PIPELINE #######################################
+
+PIPELINE_ENABLED = False
```

Comment:
@cpennington Unfortunately, I wasn't able to exactly figure out why the tests were failing. I will give it a look again.